src/plugins/discover/public/application/components/timechart_header/timechart_header.tsx

```diff
@@ -99,73 +99,78 @@ export function TimechartHeader({
   }
 
   return (
-    <I18nProvider>
-      <EuiFlexGroup gutterSize="s" responsive justifyContent="center" alignItems="center">
-        <EuiFlexItem grow={false}>
-          <EuiToolTip
-            content={i18n.translate('discover.howToChangeTheTimeTooltip', {
-              defaultMessage: 'To change the time, use the global time filter above',
+    <EuiFlexGroup
+      className="dscTimeChartHeader"
+      gutterSize="s"
+      responsive={false}
+      wrap
+      justifyContent="center"
+      alignItems="center"
+    >
+      <EuiFlexItem grow={false} className="eui-hideFor--m">
+        <EuiToolTip
+          content={i18n.translate('discover.howToChangeTheTimeTooltip', {
+            defaultMessage: 'To change the time, use the global time filter above',
```

Comment:
We try to avoid using "above" for accessibility reasons.  Can the text be:

To change the time, use the global time filter. (add ending period).

Reply:
Makes sense, changed that!